eslint.config.js

```diff
@@ -0,0 +1,57 @@
+const tseslint = require('typescript-eslint');
+const eslint = require('@eslint/js');
+const prettierPluginRecommended = require('eslint-plugin-prettier/recommended');
+const typescriptEslint = require('@typescript-eslint/eslint-plugin');
+const globals = require('globals');
+
+module.exports = tseslint.config(
+  eslint.configs.recommended,
+  tseslint.configs.recommended,
+  prettierPluginRecommended,
```

Comment:
By importing `eslint...prettier/recommended` and including it here, both the plugin and config get set up in one go, meaning we don't need to also also add the plugin to `plugins` below. See [here](https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#configuration-new-eslintconfigjs) for more info.